.github/workflows/_new_release_notification.yml

```diff
@@ -0,0 +1,24 @@
+name: "New Release Notification"
+
+on:
+  workflow_call:
+    inputs:
+      message:
+        type: string
+        required: true
+
+jobs:
+  discord:
+    runs-on: ubuntu-latest
+    env:
+      DISCORD_WEBHOOK: ${{ secrets.NEW_RELEASE_DISCORD_WEBHOOK }}
+    steps:
+      - name: Notify Discord
+        run: |
+          export TESTED_WITH_DAGGER_VERSION=0.9.10
+
+          curl --fail --silent --show-error https://dl.dagger.io/dagger/install.sh \
+          | DAGGER_VERSION=$TESTED_WITH_DAGGER_VERSION sh
```

Comment:
Any chance we could dog food https://github.com/marketplace/actions/dagger-for-github with this?

Reply:
LGTM!
<img width="1440" alt="image" src="https://github.com/dagger/dagger/assets/3342/590c910b-d06d-4bac-ab9a-b2fac33f2892">